src/SFA.DAS.AssessorService.Application.Api/Services/UkrlpProcessingService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using SFA.DAS.AssessorService.Api.Types.Models.UKRLP;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SFA.DAS.AssessorService.Application.Api.Services
+{
+    public class UkrlpProcessingService: IUkrlpProcessingService
+    {
+        public UkrlpProviderDetails ProcessDetails(List<ProviderDetails> providerDetails)
+        {
+            if (providerDetails== null || providerDetails.Count==0)
+                return new UkrlpProviderDetails();
+
+            var providerDetail = new UkrlpProviderDetails();
+
+            var firstResult = providerDetails.ToList().FirstOrDefault();
+            providerDetail.LegalName = firstResult.ProviderName;
+            if (firstResult.ProviderAliases != null && firstResult.ProviderAliases.Count > 0)
+            {
+                providerDetail.TradingName = firstResult.ProviderAliases.First().Alias;
+            }
+
+            if (firstResult.VerificationDetails == null) return providerDetail;
+
+            foreach (var verificationDetail in firstResult.VerificationDetails)
+            {
+                if (verificationDetail.VerificationAuthority?.ToLower() == "companies house")
```

Comment:
Just wondered if this, and the text searched for charity commission, should be constants, not string literals?